crates/bevy_text/src/text.rs

```diff
@@ -154,5 +154,5 @@ impl Default for TextStyle {
 #[derive(Component, Default, Copy, Clone, Debug, Reflect)]
 #[reflect(Component)]
 pub struct Text2dSize {
-    pub size: Size,
+    pub size: Vec2,
```

Comment:
I am unsure if I like the change away from `Size`. The more strongly-typed API is nice, and we can implement specialty methods on it later. Also named structs > tuple structs for clarity.

Other opinions?

Reply:
Given that `Size` still exists, I would be in favor of changing this back.

Reply:
Oh I see: this would pull in another dependency. Okay sure, that's totally fine with me then.

Reply:
The following was said by @mockersf:

>There is a pretty similar situation with the Size struct in bevy_math. It's only used as a generic Vec2, and that's only needed in bevy_ui as it's needed to have a Vec2 over Val, which is not possible with the Vec2 from glam. Most other places in code that need a size use directly a Vec2.

> I think the Size struct should move to bevy_ui and not be generic anymore. Other place that needs a size should use a Vec2 instead as it's way more powerful.

And I agree to these suggestions, because otherwise we would have to add dependencies to `bevy_ui` everywhere we want to use the `Size` type. Because the `Size` type is literally just a generic and simpler version of a `Vec2` I went with using a `Vec2` instead. 

I kept every occurrence of `Size` inside of `bevy_ui` and only changed the ones outside of that crate. This is because we need a `Vec2` of type `Val` which is only possible with the `Size` struct. The other places don't need this functionality which is why I changed them to be `Vec2`s instead. 

One thing I just noticed is that he also mentioned that the struct shouldn't be generic anymore. I will change this as well.